lighthouse-core/formatters/partials/table.html

```diff
@@ -7,15 +7,15 @@
     <thead>
       <tr>
         {{#each ../tableHeadings}}
-          <th {{#if_eq @key "preview"}}class="preview-image"{{/if_eq}}>{{this}}</th>
+          <th class="table-column table-column-{{ kebabCase @key }}">{{this}}</th>
         {{/each}}
       </tr>
     </thead>
     <tbody>
       {{#each rows}}
         <tr>
           {{#each cols}}
-            <td>{{ sanitize this}}</td>
+            <td class="table-column table-column-{{ kebabCase (lookup ../../headingKeys @index) }}">{{ sanitize this}}</td>
```

Comment:
don't like this, but not sure it's worth restructuring the format of rows just for class names here